app/controllers/api/v2/restful_controller.rb

```diff
@@ -45,7 +45,7 @@ def accessible_records
       end
 
       def current_user
-        super || token_user || doorkeeper_user || nil
+        token_user || super || doorkeeper_user || nil
```

Comment:
This way if you're logged in but have an access token, you can override the current_user when using the browser to access the api. mostly for me but i can see it being useful for other developers